### ISSUE TITLE
[Monitoring v2] Adding Windows Server 2022 support for rancher-monitoring

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#35595]
We need to move the fix in [this PR](https://github.com/rancher/charts/pull/1621) to [the monitoring rebase PR](https://github.com/rancher/charts/pull/1567) which will be released as the next version.

**Comment by jiaqiluo** [comment]
Can be tested in the rancher-monitoring  `100.1.0+up19.0.3`

Please refer to this issue for more information: https://github.com/rancher/windows/issues/92

**Comment by sowmyav27** [comment]
Blocked by - https://github.com/rancher/rancher/issues/35851

**Comment by slickwarren** [comment]
blocked by https://github.com/rancher/windows/issues/104#issuecomment-995369120

**Comment by sowmyav27** [comment]
as per @sirredbeard "This appears to be related to a breaking changes made in the Calico chart. " Moving this out of 2.6.3

**Comment by slickwarren** [comment]
should be unblocked on this, upstream fixes from calico should be resolved

**Comment by slickwarren** [comment]
blocked on https://github.com/rancher/rancher/issues/36763

**Comment by sirredbeard** [comment]
Not RKE2 Windows option.
RKE2 with Windows enabled didn't allow chart to install.
Tried just Windows options, failed
Will be reviewed today on a new setup.
Per @slickwarren 

Need to confirm on RKE2
Should automatically detect Windows and deploy daemonSets for Windows

May need to follow-up with Ross and Arvind

**Comment by slickwarren** [comment]
I'm getting this error when installing monitoring v2 v100.1.1+up19.0.3  on rancher v2.6-head (b7c9b89)
```
Error: podsecuritypolicies.policy "pushprox-kube-etcd-client" already exists
```
I've tried the following: 
* rke2 option, explicitly enabling global.cattle.windows.enabled -- results in error above
* rke2 option, leaving all defaults -- results in error above
* windows option -- installs, but some of the pods fail (this is the rke1 option so sort of expected this not to work)

I have 2 requests:
* address the error above so that monitoring will install on rke2 + windows
* clarify the `cluster options` for the user. I think that the following would help:

**Comment by jiaqiluo** [comment]
Unassigned myself and assigned it to @aiyengar2  to follow up on the issue

**Comment by aiyengar2** [comment]
https://github.com/rancher/dashboard/issues/5490

**Comment by aiyengar2** [comment]
https://github.com/rancher/dashboard/issues/5491

**Comment by aiyengar2** [comment]
UI Blocker: https://github.com/rancher/dashboard/issues/5492

**Comment by slickwarren** [comment]
bumping to 2.6.5 due to rke2 not GA this release, and will not be able to fix/validate this before release of 2.6.4.

**Comment by aiyengar2** [comment]
To summarize, here are all the issues we discovered with Windows x RKE2 x Windows LTSC 2022:

### Dashboard issues (should affect all RKE2 windows clusters)

Currently, on initial install and **first** edit/upgrade, we are seeing issues with the settings that the UI is applying for RKE2 Windows clusters:
- https://github.com/rancher/dashboard/issues/5490
- https://github.com/rancher/dashboard/issues/5491
- https://github.com/rancher/dashboard/issues/5492

**Workaround: always manually enable all the `rke2*.enabled` and `global.cattle.windows.enabled` and manually disable all `rke*.enabled` flags every time you install or do an edit/upgrade till UI resolves the issue.**

### Kubelet issue (should only affect Windows LTSC 2022)

kubelet was not being deployed due to issues with NodePorts in 2022 that is making it impossible to hit ${NODE_IP}:10252/metrics on the hosts to collect metrics: possibly related to upstream issues in https://github.com/microsoft/Windows-Containers/issues/204 and noted in https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#load-balancing-and-services

**Conclusion: We should not be testing Windows LTSC 2022 in RKE1 or RKE2 till upstream issues are resolved** 

cc: @deniseschannon @sowmyav27 

### ingress-nginx is not up (possibly only affects Windows LTSC 2022?)

Unsure. @slickwarren is going to see if this is only related to Windows LTSC 2022 before we investigate any further; could be due to the same issue we saw in kubelet above

### windows-exporter, wins-upgrader issues

Tracked in https://github.com/rancher/rancher/issues/37020 and https://github.com/rancher/rancher/issues/37022; the TLDR is that the path `c:\Windows\wins.exe` only applies for RKE1 clusters, not RKE2 clusters, so @luthermonson is going to fix these.

**Comment by aiyengar2** [comment]
unassigning myself from this issue; feel free to assign me back or reach out to me if I need to take a look at anything.

**Comment by sirredbeard** [comment]
Need to dev validate and then move to to test.

**Comment by rosskirkpat** [comment]
This is blocked by https://github.com/rancher/rancher/issues/37022 and https://github.com/rancher/rancher/issues/37020, which @luthermonson is actively working on.

**Comment by sirredbeard** [comment]
This issue requires a testing template and needs to be moved into test ASAP @luthermonson

**Comment by sirredbeard** [comment]
Closing in favor of #37022
